crates/cargo-wdk/tests/build_command_test.rs

```diff
@@ -77,16 +73,9 @@ fn emulated_workspace_builds_successfully() {
         let emulated_workspace_path = "tests/emulated-workspace";
         let stdout = run_build_cmd(emulated_workspace_path);
 
-        // Matches warning about WDK metadata not being available for non driver project
-        // but a valid rust project
-        assert!(stdout.contains(
-            "WDK metadata is not available. Skipping driver build workflow for package: \
-             rust-project"
-        ));
-
-        assert!(stdout.contains("Processing completed for package: driver_1"));
-        assert!(stdout.contains("Processing completed for package: driver_2"));
-        assert!(stdout.contains(r"Build completed successfully"));
+        assert!(stdout.contains("Building package driver_1"));
```

Comment:
Not to be tackled in this PR, but i wonder if this is better suited to be tested via snapshot testing. I've heard good things about trycmd: https://docs.rs/trycmd/latest/trycmd/index.html

Reply:
Absolutely. We should try this approach.